props/saber_BC_buttons.h

```diff
@@ -748,12 +804,17 @@ class SaberBCButtons : public PROP_INHERIT_PREFIX PropBase {
     case EVENTID(BUTTON_POWER, EVENT_FIRST_SAVED_CLICK_SHORT, MODE_OFF):
       // No power on without exiting Vol Menu first
       if (!mode_volume_) {
-        On();
+      // Bypass preon if pointing up  
+        if (fusor.angle1() >  M_PI / 3) {
+          SaberBase::TurnOn();
```

Comment:
This skips several steps that On() does, not sure if any of those are actually required though....

Reply:
That's was the idea. Skip preon and no font.wav played, just out.wav on the live switch of presets (and I guess FastOn too if that exists.) Not sure what faston.wav is supposed to be if we're already playing a normal ignition sound, but this works great for me.

Reply:
You might want to respect NoOnOff() at least.

Reply:
Updated to just call FastOn() instead now.